tests/test_calc_vector.py

```diff
@@ -39,20 +39,21 @@ def test_calc_vector_higher_than_allowed_missing_percentage():
 
 def test_calc_vector_within_allowed_missing_percentage():
     """Test using a pretrained network and a missing word percentage within allowed."""
-    spectrum = Spectrum(mz=numpy.array([100, 111.1, 200, 250], dtype="float"),
+    spectrum = Spectrum(mz=numpy.array([11.1, 100, 200, 250], dtype="float"),
                         intensities=numpy.array([0.1, 0.1, 0.1, 1.0], dtype="float"),
                         metadata={})
 
     document = SpectrumDocument(spectrum, n_decimals=1)
     model = import_pretrained_model()
     vector = calc_vector(model, document, intensity_weighting_power=0.5, allowed_missing_percentage=17.0)
-    expected_vector = numpy.array([-0.0760859, -0.09670882, 0.10740811, 0.04696679, 0.00893024,
-                                   -0.04496526, -0.0079267, -0.05153507, -0.00433566, -0.06211582])
-    assert document.words[1] not in model.wv.vocab, "Expected word to be missing from given model."
+    expected_vector = numpy.array([0.12775915, -1.17673617, -0.14598507, -0.40189132, 0.36908966,
+                                   1.11608575, -1.46774333, -0.31442554, -0.23168877, -0.29420064])
+    assert document.words[0] not in model.wv.vocab, "Expected word to be missing from given model."
     assert numpy.all(vector == pytest.approx(expected_vector, 1e-5)), "Expected different document vector."
 
 
 def import_pretrained_model():
```

Comment:
Use [pytest fixture](https://docs.pytest.org/en/latest/fixture.html) instead.

```py
import pytest

def test_calc_vector_higher_than_allowed_missing_percentage(model):
    ...

@pytest.fixture(scope="function")
def model():
    ...
    return gensim.models.Word2Vec.load(model_file)
```

Reply:
I tried it but then ran into ``AttributeError: 'function' object has no attribute 'wv'``. So I would (for time sake) leave it as it is if that is OK?